frontend/pages/investor/[id]/index.tsx

```diff
@@ -0,0 +1,192 @@
+    website: 'https://www.site.com',
+    social: [
+      { id: 'linked-in', url: 'https://www.linkedin.com' },
+      { id: 'twitter', url: 'https://www.twitter.com' },
+      { id: 'facebook', url: 'https://www.facebook.com' },
+      { id: 'instagram', url: 'https://www.instagram.com' },
+    ],
+    contact: 'LoÏc Comolli',
+  };
+
+  const tagsGridRows: TagsGridRowType[] = [
+    {
+      title: 'Invests in',
+      type: 'category',
+      tags: [
+        { id: 'tourism', title: 'Tourism & Recreation' },
+        { id: 'production', title: 'Non-timber forest production' },
+        { id: 'agrosystems', title: 'Sustainable agrosystems' },
+        { id: 'forestry', title: 'Forestry & agroforestry' },
+      ],
+    },
+    {
+      title: 'Ticket size',
+      tags: ['US$50k', '$50k - $500k', '$500k - $1M'],
+    },
+    {
+      title: 'Instrument size',
+      tags: ['Grand', 'Loan'],
+    },
+    {
+      title: 'Impact they invest on',
+      tags: ['Biodiversity', 'Community'],
+    },
+  ];
+
+  const sdgsItems: SDGType[] = [
+    { id: 'no-poverty', title: 'No poverty' },
+    { id: 'gender-equality', title: 'Gender equality' },
+    { id: 'decent-work', title: 'Decent work and economic growth' },
+    { id: 'reduced-inequalities', title: 'Reduced inequalities' },
+    { id: 'climate-action', title: 'Climate action' },
+    { id: 'life-on-land', title: 'Life on land' },
+  ];
+
+  return (
+    <>
+      <Head title="Investor Profile" />
+
+      <ProfileHeader
+        logo="/images/temp-placeholders/nesst-logo.png"
+        title="NESst"
+        subtitle="Non-VC Investment vehicle"
+        text={aboutInfo.text}
+        website={aboutInfo.website}
+        social={aboutInfo.social}
+        contact={aboutInfo.contact}
+      />
+
+      <LayoutContainer layout="narrow" className="mt-24 mb-20 md:mt-40">
+        <section aria-labelledby="profile-investment-info">
+          <h2
+            id="profile-investment-info"
+            className="mt-12 font-serif text-2xl font-semibold md:mt-20 sm:text-3xl text-green-dark"
+          >
+            <FormattedMessage defaultMessage="Investment info" id="m3Dnav" />
+          </h2>
+
+          <TagsGrid className="mt-10 md:mt-14" rows={tagsGridRows} />
+
+          <h3 className="mt-10 mb-3 text-xl font-semibold md:mt-14">
+            <FormattedMessage defaultMessage="Invests in SDG's" id="7qgtEX" />
+          </h3>
+          <SDGs className="my-3" sdgs={sdgsItems} />
+
+          <h3 className="mt-10 mb-3 text-xl font-semibold md:mt-14">
+            <FormattedMessage defaultMessage="How do they work?" id="KtecFi" />
+          </h3>
+          <p className="my-3">
+            We achieve our mission through an Incubator that provides pre-seed capital and business
```

Comment:
Shouldn't you store this text in a variable as well? It'll come from the API, right?

I'm ok to keep this temporarily. I'm just checking if it's hard-coded text that should be translated.

Reply:
The way I'm going about it, is to use `intl` on everything that I recognise will be translated in the frontend; everything else I expect to come from the API. 

I don't think it's worth it to throw everything into variables because we'd most likely end up renaming and reorganising everything when we connect to the API; we don't know yet data formats/properties names. Some content I did put on variables just for layout development convenience/reusability purposes (mocked data I'm looping through to build views,  or use in other screens, or pass to components).